eng/Packages.Data.props

```diff
@@ -57,6 +57,29 @@
   <!--
     Dependency versions for Track 2, Azure.* libraries.
     Only packages that are approved dependencies should go here.
+    Contact azuresdkengsysteam@microsoft.com for approval.
```

Comment:
I don't think they should be contacting EngSys for these approvals. That should be handled by the language team or the architects depending on which reference we are talking about.

Reply:
Looks like that was copied from the contributing which I also don't believe is correct.

Reply:
Any idea why that's the address in the existing blurb in the contrib guide?  I agree that it doesn't seem right that it's EngSys.

Reply:
I couldn't find any specific guidance for approval of dependencies in the docs/wiki/engineering hub, maybe the best point of contact is just our team? Maybe azsdknet@microsoft.com?

Reply:
Let's drop the email entirely and just refer whomever is attempting to make the change to "seek architect approval."    We should put the Core team members as well as the architects as owners of the packages file in CODEOWNERS to make sure that things don't sneak in.

Reply:
That makes sense to me. I can make a separate PR to propose the update to CODEOWNERS

Reply:
Here is the PR : https://github.com/Azure/azure-sdk-for-net/pull/41039

Reply:
>Any idea why that's the address in the existing blurb in the contrib guide?

Probably legacy and we should probably also update that.